cmd/snaplock/runinhibit/inhibit.go

```diff
@@ -104,9 +107,10 @@ func removeInhibitInfoFiles(snapName string) error {
 	if err != nil {
 		return err
 	}
+	hintFile := fmt.Sprintf("%s.%s", snapName, hintFilePostfix)
```

Comment:
my suggestion was a bit different to avoid repeating this logic here, hintFile := fiilepath.base(HintFile(snapName))

Reply:
yes, my bad. thank you for pointing it out.